cpp/src/aztec/honk/sumcheck/polynomials/multivariates.test.cpp

```diff
@@ -110,8 +112,194 @@ TYPED_TEST(MultivariatesTests, FoldTwoRoundsGeneric)
 
     FF round_challenge_1 = FF::random_element();
     FF expected_val = expected_lo * (FF(1) - round_challenge_1) + expected_hi * round_challenge_1;
-
     multivariates.fold(multivariates.folded_polynomials, multivariate_n >> 1, round_challenge_1);
     EXPECT_EQ(multivariates.folded_polynomials[0][0], expected_val);
 }
+
+// Three rounds
+/*f0(X1, X2, X3) = v000(1-X1)(1-X2)(1-X3) + v001(1-X1)(1-X2)X3 + v010(1-X1)X2(1-X3) + v011(1-X1)X2X3
```

Comment:
Looks correct. Could you also align this a bit more clearly so it's easier to read?

Reply:
Done.